resources/aws/launch_configuration.go

```diff
@@ -22,6 +23,13 @@ type LaunchConfiguration struct {
 	Client *autoscaling.AutoScaling
 }
 
+const (
+	defaultEBSVolumeMountPoint = "/dev/xvdh"
+	// defaultEBSVolumeSize is expressed in GB.
+	defaultEBSVolumeSize = 50
+	defaultEBSVolumeType = "gp2"
+)
```

Comment:
As you already mentioned in your PR description, I think this should come from the TPO at some point. Kubernetesd can be configured with a mapping of these information according to the used EC2 instance type, where these information are bound to, if I understood it correctly. Lets please create an issue if not already existing.